webextensions/api/devtools.json

```
@@ -479,7 +479,7 @@
                 "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools/panels/ExtensionSidebarPane/setExpression",
                 "support": {
                   "chrome": {
-                    "version_added": "≤61",
+                    "version_added": "19",
```

:x: Same as `inspectedWindow`

```suggestion
                    "version_added": "18",
```